app/containers/Questions/Content/Body/TopQuestion.js

```diff
@@ -2,11 +2,14 @@ import React, { memo, useCallback, useMemo, useState } from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
 
-import topQuestionActiveIcon from 'images/starActive.svg?inline';
-import topQuestionsInactiveIcon from 'images/star.svg?inline';
+import topQuestionActiveIcon from 'images/starActive.svg?external';
+import topQuestionsInactiveIcon from 'images/star.svg?external';
 
 import TopQuestionPopover from './TopQuestionPopover';
+import { Icon20 } from 'components/Icon/IconWithSizes';
```

Comment:
I would not recommend include specifics into component names or styles. If size of this icon needs to change to 21 then name will not makes sense and you will have to rename it everywhere.

It is better to give logical names. E.g. IconXs, IconSm, IconMd, IconLg, etc

Reply:
fixed it